src/ralph/admin/sites.py

```diff
@@ -0,0 +1,136 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import six
+
+from django.apps import apps
+from django.conf.urls import url
+from django.contrib.admin.sites import AdminSite
+from django.core.urlresolvers import NoReverseMatch, reverse
+from django.template.response import TemplateResponse
+from django.utils.text import capfirst
+
+
+def get_urls_chunks(model, view):
+    """
+    Returns a tuple to generate a URL for an additional view
+
+    :Example:
+
+        >>> "{}_{}_{}".format(*get_urls_chunks(model, view))
+        app_label_model_name_url_name
+
+    :param model: Django admin model
+    :type model: models.Model
+    :param view: Django generic view object
+    :type view: GenericView
+
+    :return: tuple of names
+    :rtype: tuple
+    """
+
+    return (
+        model._meta.app_label, model._meta.model_name, view.url_name
+    )
+
+
+class RalphAdminMixin(object):
```

Comment:
there are 2 `RalphAdminMixin`s - one here and one in `admin/mixins.py` - it may be misleading so could you rename it somehow?